tests/MyGame/Example2/Monster.go

```diff
@@ -9,17 +9,26 @@ import (
 type MonsterT struct {
 }
 
+// MonsterT object pack function
 func (t *MonsterT) Pack(builder *flatbuffers.Builder) flatbuffers.UOffsetT {
-	if t == nil { return 0 }
+	if t == nil {
+		return 0
+	}
+
+	// pack process all field
```

Comment:
Why are changes to this function in this PR? what does this comment mean?

Reply:
it's should be remove.